units/sdl2_mixer.pas

```diff
@@ -161,7 +153,9 @@   TMix_Chunk = record
     MUS_MP3_MAD_UNUSED,
     MUS_FLAC,
     MUS_MODPLUG_UNUSED,
-    MUS_OPUS
+    MUS_OPUS,
+    MUS_WAVPACK,
+    MUS_GME
```

Comment:
Apllies for `TMIX_Fading` and `TMIX_MusicType`:

This is not how to translate `typedef enum` according to our rules. There should be the type def. and then the entries should be constants covering with typed alias.

```
type
  PPSDL_JoystickPowerLevel = ^PSDL_JoystickPowerLevel;
  PSDL_JoystickPowerLevel = ^TSDL_JoystickPowerLevel;
  TSDL_JoystickPowerLevel = type Integer;

const
  SDL_JOYSTICK_POWER_UNKNOWN = TSDL_JoystickPowerLevel(-1);
  SDL_JOYSTICK_POWER_EMPTY   = TSDL_JoystickPowerLevel(0);  {* <= 5% *}
  SDL_JOYSTICK_POWER_LOW     = TSDL_JoystickPowerLevel(1);
```

Reply:
Done.